components/InputBox/index.tsx

```diff
@@ -143,16 +168,29 @@ const InputBox = ({
   useEffect(() => {
     if (!isUploadScreen) {
       setMessage('');
+      setFormattedMessage('');
       setInputHeight(25);
     }
   }, [fileSent]);
 
   // to clear message on ChatScreen InputBox when fileSent from UploadScreen
   useEffect(() => {
     if (isEditable) {
-      setMessage(selectedMessages[0]?.answer);
+      let convertedText = convertToMentionValues(
+        editMessage?.answer,
+        ({URLwithID, name}) => {
+          // this is used to extract ID from route://member/4544 from this kind if url
+          const match = URLwithID.match(routeRegex);
+          if (name === '@participants' || name === '@everyone') {
```

Comment:
why hard checks for group tags, if we create more tags in future, then we need a separate release for ReactNative as well.